config/authorities/linked_data/oclc_fast.json

```diff
@@ -28,21 +26,22 @@
       "id_ldpath":       "dcterms:identifier",
       "label_ldpath":    "skos:prefLabel",
       "altlabel_ldpath": "skos:altLabel",
-      "sameas_ldpath":   "schema:sameAs"
+      "sameas_ldpath":   "rdfs:sameAs"
     }
   },
   "search": {
     "url": {
       "@context": "http://www.w3.org/ns/hydra/context.jsonld",
       "@type": "IriTemplate",
-      "template": "http://experimental.worldcat.org/fast/search?query={subauth}+all+%22{query}%22&sortKeys=usage&{?maximumRecords}",
+      "template": "http://experimental.worldcat.org/fast/search?query={subauth}+all+%22{query}%22&sortKeys=usage&maximumRecords={maximumRecords}",
```

Comment:
This change means that maximumRecords will always be part of the generated URL and it will use the default if not provided.